src/hooks/mongodb4x.test.js

```diff
@@ -0,0 +1,109 @@
+import { HandlerInputsBuilder } from '../../testUtils/handlerInputsBuilder';
+import { getMockedMongoClient, wrapMongoCollection } from '../../testUtils/mongo4xMocker';
+import { MongoSpanBuilder } from '../../testUtils/mongoSpanBuilder';
+import { SpansContainer, TracerGlobals } from '../globals';
+import { hookMongoDb } from './mongodb';
+
+const DUMMY_URL = 'mongodb://localhost:27017/myproject';
+
+const validSimpleFlowArguments = [
+  [[DUMMY_URL]],
+  [[DUMMY_URL, {}]],
+  [[DUMMY_URL, { url: DUMMY_URL }]],
+  [[DUMMY_URL, undefined]],
+];
+
+const invalidSimpleFlowArguments = [[[0, 1]], [[0, 'abc']], [[0, () => {}]]];
+
+describe('mongodb', () => {
+  let connection;
+
+  beforeEach(() => {
+    const handlerInputs = new HandlerInputsBuilder().build();
+    TracerGlobals.setHandlerInputs(handlerInputs);
+  });
+
+  afterEach(() => {
+    connection && connection.close();
+  });
+
+  test.each(validSimpleFlowArguments)('hookMongoDb -> simple flow', async (args) => {
```

Comment:
```suggestion
  test.each(validSimpleFlowConstractorArguments)('hookMongoDb -> simple flow', async (args) => {
```

Reply:
Don't we need to change something in the test name? I remember something that @vladlLumigo showed me.
@vladlLumigo Can u comment here?

Reply:
so much better! thanks both of you 👍